tests/vnc_test.go

```diff
@@ -240,6 +242,21 @@ var _ = Describe("[rfe_id:127][crit:medium][arm64][vendor:cnv-qe@redhat.com][lev
 				return nil
 			}, 60*time.Second).ShouldNot(HaveOccurred())
 		})
+
+		It("should allow creating a VNC screenshot in PNG format", func() {
+			filePath := filepath.Join(GinkgoT().TempDir(), "screenshot.png")
+
+			cmd := clientcmd.NewVirtctlCommand("vnc", "screenshot", "--namespace", vmi.Namespace, "--file", filePath, vmi.Name)
+			Expect(cmd.Execute()).To(Succeed())
+
+			f, err := os.Open(filePath)
```

Comment:
Probably `defer f.close()` is needed here?

Reply:
Yep. :+1:

Reply:
Done.